server/dao/issues.js

```diff
@@ -1,28 +1,36 @@
 const Issue = require("../models/issue");
 
-function getIssuesByBoardId(id, callback){
-    Issue.find({issue_id:id}, (err, result) => {
-        callback && callback(err, result);
-    });
+function createIssue(issue, userId, columnId, callback) {
+    Issue.create({...issue, reporter_id:userId, column_id:columnId}, (createIssueErrors, createIssue)=>{
+        callback && callback(createIssueErrors,createIssue)
+    })
 }
 
-function createIssue(issue, callback) {
-    Issue.create(issue, (err, result) => {
-        callback && callback(err, result);
+function getIssuesByColumnId(id, callback){
+    Issue.find({ column_id:id }, (foundIssuesErrors, [foundIssues]) => {
```

Comment:
Unnecessary array destructing. In this case, you need to return all collection, not only the first element.